relay-server/src/endpoints/common.rs

```diff
@@ -346,6 +346,8 @@ fn check_envelope_size_limits(config: &Config, envelope: &Envelope) -> bool {
                 attachments_size += item.len()
             }
             ItemType::Session => session_count += 1,
+            // TODO: extract the actual session counts from the aggregates
```

Comment:
```suggestion
```

This TODO can be resolved. Quantity for sessions is defined as the number of batches, not the number of sessions.

Reply:
But I still need to parse this here?

Reply:
No, you can just assume that every session aggregate counts as 1. Quantity on sessions doesn't have a meaning until we emit outcomes for them, which we're not planning in the immediate future.